.github/workflows/desktop-build.yml

```diff
@@ -210,25 +210,22 @@ jobs:
             make_package: 1
 
           - target: 14
-            soc: Apple
```

Comment:
I disagree with removing these, even though the only value being checked is Intel, setting the others to Apple provides useful information

Reply:
I'm not convinced keeping the Apple tag does provide useful information. It's the new default architecture/build and will replace Intel on Macs/macOS.
Hence, only labeling the unusual & special build (Intel) makes sense in my opinion.

Same with Windows and 32/64bit. x64 was standard, so we labelled only the non-standard build (32bit) later.
Now, there is only 64bit and we do not need to label it on top.

In the CI builds and artifacts we are not showing the Apple label anymore already:
![image](https://github.com/user-attachments/assets/e549d5b0-16d9-4be1-9627-a1def54542f8)